docs/content/latest/admin/yb-admin.md

```diff
@@ -1515,6 +1515,38 @@ yb-admin \
     list_cdc_streams
 ```
 
+#### delete_cdc_stream <stream_id> [force_delete]
+
+Deletes underlying CDC stream for the specified YB-Master servers.
+
+**Syntax**
+
+```sh
+yb-admin \
+    -master_addresses <master-addresses> \
+    delete_cdc_stream <stream_id [force_delete]>
+```
+
+* _master-addresses_: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`.
+* *stream_id*: The ID of the CDC stream.
+* `force_delete`: (Optional) Force the delete operation.
+
+#### bootstrap_cdc_producer <comma_separated_list_of_table_ids>
+
+Mark a set of tables in preparation for setting up universe level replication.
```

Comment:
I guess as a future todo, we should link this back to wherever we add the steps for doing the full setup process with bootstrapping.

For now, do we need to explain more here as to what this is doing?

Reply:
I think we should at least mention that this returns a list of bootstrap ids / stream ids, that should be passed on to the corresponding setup/alter_universe_replication command after the backup/restore

Reply:
apparently we didn't even list those in the setup/alter, so I'm adding those in and doing some x-reference